test-exports/index.js

```diff
@@ -1,6 +1,6 @@
 const assert = require("assert");
-const { OpenAI } = require("langchain");
-const { loadPrompt } = require("langchain/prompts");
+const { OpenAI } = require("../langchain/dist");
```

Comment:
IMO, that's the safest way of testing the imports here. We build it first (build is a dependency now, see `turbo.json`) and then test the imports directly.

Reply:
The intent here is to test exactly what is in `exports` field in package.json, doing it like this will skip that I believe?

Reply:
makes sense. I changed that